### PR TITLE
Q.notice

### PR DESCRIPTION
**Pull request by AndreyTepaykin**
Added Q.Notice API to Q.js and styles to Q.css
Removed old Q.Notice api and it's traces.

### CODE REVIEW

File: platform/plugins/Q/web/js/Q.js [hunk 1]
```diff
@@ -12651,6 +12651,175 @@ Q.Camera = {
...
+	/**
+	 * Adds a notice.
+	 * @method add
+	 * @param {object} options Object of options
+	 * @param {string} [options.key] Unique key for this notice. Need if you want to modify/remove notice by key.
+	 * @param {string} options.content HTML contents of this notice.
+	 * @param {bool} [options.closeable=true] Whether notice can be closed with red x icon.
+	 * @param {function|string} [options.handler] Something (callback or URL) to handle with Q.handle()
+	 * @param {string} [options.type=common] Arbitrary type of notice. Can be used to apply different styles dependent on type,
+	 * @param {bool|number} [options.timeOut=false] Time in seconds after which to remove notice.
```
Comment (EGreg) [inline platform/plugins/Q/web/js/Q.js]: `timeout` not `timeOut`

File: platform/plugins/Q/web/js/Q.js [hunk 2]
```diff
@@ -12651,6 +12651,175 @@ Q.Camera = {
...
+	 * Container for notices
+	 * @property container
+	 * @type {HTMLElement}
+	 */
+	container: document.getElementById("notices_slot"),
+
+	/**
+	 * Adds a notice.
+	 * @method add
+	 * @param {object} options Object of options
```
Comment (EGreg) [inline platform/plugins/Q/web/js/Q.js]: Just make all the names uppercase, like `{Object}`

**Review by EGreg — CHANGES_REQUESTED** [review]
Tiny changes

**Review by EGreg — APPROVED** [review]

File: platform/plugins/Q/web/js/Q.js (hunk 1 above)
Comment (EGreg) [inline platform/plugins/Q/web/js/Q.js]: Please fix case:`timeOut` should be `timeout` and all `{foo}` should be `{Foo}`

**Review by EGreg — CHANGES_REQUESTED** [review]
Tiny changes

**Review by EGreg — APPROVED** [review]

File: platform/plugins/Q/web/js/Q.js (hunk 1 above)
Comment (AndreyTepaykin) [reply platform/plugins/Q/web/js/Q.js]: Done.

File: platform/plugins/Q/web/js/Q.js (hunk 2 above)
Comment (AndreyTepaykin) [reply platform/plugins/Q/web/js/Q.js]: Done

File: platform/plugins/Q/web/js/Q.js (hunk 1 above)
Comment (AndreyTepaykin) [reply platform/plugins/Q/web/js/Q.js]: Done. Please look last commit.

**Review by EGreg — APPROVED** [review]

**Pull request merged**